README.md

```diff
@@ -428,25 +422,22 @@ If using Rinkeby, you can verify the transaction was sent using
 
 ![Etherscan display of transaction on Rinkeby](static/etherscan-send-tx.png)
 
-## Multitenancy
+## Advance Tutorial
+
+### Multitenancy
 
-Multi-tenancy enables serving of multiple blockchain applications with a single Orchestrate instance.
-Resources including transaction streams, access to the blockchain network, private keys,
-and smart contracts are isolated to the tenant that owns them.
+Multi-tenancy enables serving of multiple blockchain applications with a single Orchestrate instance. Resources including transaction streams, access to the blockchain network, private keys, and smart contracts are isolated to the tenant that owns them.
```

Comment:
enables serving multiple blockchain applications

Reply:
I didn't modify that, but I look better as you write it. Updated!